boltdesignsystem/bolt
# DS-801 Flag Object Deprecation

## Jira

https://pegadigitalit.atlassian.net/browse/DS-801

## Summary

The `flag object` is deprecated. There are no instances in the Bolt repository of utilizing this object anymore.

## Details

The `flag object` is deprecated. All examples of its usage were replaced with the `Layout` using `template: 'flag'`

## How to test

Pull the branch. Make sure there is no usage of the `flag object` in PL and whether replacing the `flag object` with the `Layout, template: 'flag'` makes no differences and regressions.

## Release notes

The `flag object` (`@bolt/flag.twig`) is deprecated. Please use the `Layout` with the `template: 'flag'` option whenever needed.
